ethereum/migrations/7_deploy_token_implementation_only.js

```diff
@@ -0,0 +1,13 @@
+// run with:
+// npm run deploy-token-implementation-only
+// e.g. Ethereum Mainnet
+// INFURA_KEY="" MNEMONIC="" npm run deploy-token-implementation-only -- --network mainnet
+// e.g. BSC
+// MNEMONIC="" npm run deploy-token-implementation-only -- --network binance
+// e.g. Polygon
+// MNEMONIC="" npm run deploy-token-implementation-only -- --network polygon
+const TokenImplementation = artifacts.require("TokenImplementation");
```

Comment:
maybe consider adding this file as a regular truffle script, like: https://github.com/wormhole-foundation/wormhole/blob/3325e8ba02598b666a95947337a1714d6b3512f6/ethereum/scripts/deploy_token_bridge.js since the migration flow doesn't make that much sense for these one-off deploys

Reply:
Will consider adding another regular truffle script in a future PR. I think (token) bridge has both which is 🤷 . Personally I like the simulation, but agree it's not super necessary (and is weightier).